src/bigscreenplayer.js

```diff
@@ -607,6 +607,12 @@ function BigscreenPlayer () {
      */
     getLogLevels: () => DebugTool.logLevels,
 
+    /**
+     * @function
+     * @return {string} - currently set log level @see getLogLevels
+     */
+    getLogLevel: () => DebugTool.getLogLevel,
```

Comment:
This should be `DebugTool.getLogLevel()`

Reply:
In fact, I think it should just be `getLogLevel: DebugTool.getLogLevel`

To be honest, I'm not even sure it's valuable adding it to the top level API.

Reply:
That's a fair call too!

Is your thinking to export `DebugTools` in `main.js` instead?

Reply:
No, I'm just not sure if people will actually use the API outside of the BSP codebase itself to get the log level. I'll leave it just in case, not harming anything 😄